### ISSUE TITLE
a11y: incorrect/misleading/uninformative screen-reader labels

### ISSUE THREAD

**Issue by beyackle** [microsoft/BotFramework-Composer#6559]
## Describe the bug

This ticket describes the current state of the screen-reader text; there are a number of issues as noted below:

- "No item in view" on triggers in tree
- Error display just reads numbers (e.g. "6 3")
- focus moves backward when collapse/expand chevron is selected
- "one bot have been found"
- Just "button" for expand/collapse on bot project settings
- "westus" read instead of LUIS region
- Copy content for translation dialog is missing explanatory labels
- read out "Installing plugin" text from dialog
- adapter section: "Manage profiles" doesn't read ABS channels
- app settings page: toggles don't say what they do

## Version

1.4.0 

## Browser

- [ ] Electron distribution
- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [x] Edge

## OS

- [ ] macOS
- [x] Windows
- [ ] Ubuntu

**Comment by beyackle** [comment]
I think this can be closed once those two PRs merge; I've double-checked the list and the one remaining issue that looks unsolved (the "focus moves backward" one) I wasn't able to duplicate.
